site/content/en/docs/administration/basics/installation.md

```diff
@@ -444,14 +444,14 @@ If you stay in China, for installation you need to override the following source
   [Ubuntu mirroring help](https://mirrors.tuna.tsinghua.edu.cn/help/ubuntu/)
 
   Pre-compiled packages:
-  ```
+  ```bash
   deb https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ focal main restricted universe multiverse
   deb https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ focal-updates main restricted universe multiverse
   deb https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ focal-backports main restricted universe multiverse
   deb https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ focal-security main restricted universe multiverse
   ```
   Or source packages:
-  ```
+  ```bash
```

Comment:
I see several ways to mark similar code: bash, sh, console. Could you please choose one?

Reply:
@TOsmanov , could you please answer?

Reply:
in the `installation.md` file I replaced `bash` with  `sh`

Reply:
I replaced the language `console` and `sh` with `bash` in the latest version of the documentation as `bash` looks better, but css for other languages should remain in the `_custom.css` as it is used in older versions of the documentation